### PR TITLE
Update Client 12.0 to stable

### PR DESCRIPTION
**Pull request by palnabarun**
- [x] Merge changes from master
- [x] Update constants
- [x] Regenerate client
- [x] Apply hotfixes
- [x] Update CHANGELOG and README

### CODE REVIEW

File: changes.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
+6d64cf67 Merge pull request #1265 from kubernetes-client/dependabot/github_actions/actions/setup-python-v2.1.3
```
Comment (roycaihw) [inline changes.txt:1]: Is this file generated?

**Review by roycaihw — COMMENTED** [review]
LGTM overall. Thanks!

**Comment by roycaihw** [comment]
Regarding the test failure https://github.com/kubernetes-client/python-base/pull/215, can we exclude the python-base update (which changed the error behavior) in this stable release. We can fix and include it in the next alpha release.

File: changes.txt (hunk 1 above)
Comment (palnabarun) [reply changes.txt:1]: Yeah. That upgrade was done by `dependabot`. It was enabled in https://github.com/kubernetes-client/python/pull/1259.

**Comment by palnabarun** [comment]
Yay!! :tada: The tests passed. :)

**Comment by roycaihw** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**